Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -100,25 +172,52 @@ public void Generate_LicenseAcceptanceRequiredByMainPackage()
             TestLogger logger = new TestLogger();
             AnalyzerPluginGenerator apg = CreateTestSubjectWithFakeRemoteRepo(remoteRepoBuilder, logger);
 
-            // 1. User does not accept -> fails with error
-            ProcessedArgs args = CreateArgs("parent.requiredAccept.id", "1.0", "cs", null, false /* accept licenses */ , outputDir);
+            // 1. a) Only target package. User does not accept -> fails with error
```

Comment:
I'm not sure how much value the additional "recurse=false/true" cases are adding here. The existing tests checked that generation would only succeed if the licenses were acceptance, regardless of the level at which the license existed.